src/components/form/form.scss

```diff
@@ -10,12 +10,8 @@
     }
 }
 
-.limel-form-layout--grid {
+.limel-form-layout--default {
+    --gap: 1rem;
```

Comment:
Does this mean the consumer can override this by setting their own value for `--gap`?

If so, should we document that?

Reply:
This wasn't my initial intention. the `var` was there only to be used in the `calc`s. I don't think we should expose this for external tweaks, at the moment at least.

Reply:
It doesn't seem like it can be set from the outside anyway (not by just setting the value on the `limel-form` element at least). I guess we would have to define it on `:host` for that to be possible. So we're all good 👍